client/components/LearnerViewDirective.js

```diff
@@ -96,12 +96,24 @@ tie.directive('learnerView', [function() {
               <h3 class="tie-question-title">{{title}}</h3>
               <div class="tie-previous-instructions">
                 <div ng-repeat="previousInstruction in previousInstructions track by $index">
-                  <p ng-repeat="paragraph in previousInstruction track by $index">{{paragraph}}</p>
+                  <div ng-repeat="instruction in previousInstruction track by $index">
+                    <p ng-if="instruction.type == 'text'">{{instruction.content}}</p>
+                    <div ng-if="instruction.type == 'example'">
+                      Example: <pre class="tie-question-example">{{instruction.content}}</pre>
+                    </div>  
+                    <pre class="tie-question-code" ng-if="instruction.type == 'code'">{{instruction.content}}</pre>
```

Comment:
What I mean is, why not just have types 'text' and 'code' (instead of 'text', 'code', and 'example')? I don't see the value in having an 'example' type when it appears the only difference between 'example' and 'code' is that the text 'Example:' is prepended.